Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM maven:3-jdk-8-onbuild-alpine
```

Comment:
Yes, because it is `onbuild` image. 

You can see what it does https://github.com/carlossg/docker-maven/blob/322d0dff5d0531ccaf47bf49338cb3e294fd66c8/jdk-8/onbuild/Dockerfile